channels/ON-CHAIN.md

```diff
@@ -176,6 +159,8 @@ This transaction MUST have valid signatures of all involved parties.
 After this transaction has been included in a block, the channel MUST be
 considered closed and allow no further modifications.
 
+`channel initiator_amount + responder_amount == 
```

Comment:
The internal distribution of tokens inside the channel is not known on-chain so we don't really have the notion of `channel initiator_amount + responder_amount`. In the light of withdrawals and deposits, this certainly can not be the channel create balances...
Probably `channel total amount` could fit better?